### PR TITLE
Add commit hash to oracle-core version

### PR DESCRIPTION
**Pull request by SethDusek**
For #20

### CODE REVIEW

**Comment by greenhat** [comment]
@SethDusek I feel like I'm waiting for you to mark it as ready while it actually was ready a long time ago. :) Is it the case?

**Comment by SethDusek** [comment]
Yup, I actually forgot about this PR for a while. It should be good to go

File: core/src/main.rs [hunk 1]
```diff
@@ -67,8 +67,16 @@ pub type BlockDuration = u64;
 /// The epoch counter
 pub type EpochID = u32;
 
+const ORACLE_VERSION: &str = concat!(
```
Comment (greenhat) [inline core/src/main.rs:70]: I think `APP_VERSION` or even `VERSION` would be a better name since we don't have any other versions.

**Review by greenhat — CHANGES_REQUESTED** [review]
Looking great! Just a tiny note.

**Review by greenhat — APPROVED** [review]
Great! Thank you!

**Pull request merged**